docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,283 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get started with W&B Inference
+
+This guide shows you how to use W&B Weave with the W&B Inference service. You can build and trace LLM applications without setting up your own infrastructure or managing API keys from multiple providers.
+
+The W&B Inference service helps you learn Weave faster because:
+- You can start immediately without obtaining API keys from other providers
+- You get access to multiple models in one place, making it easier to compare their behavior
+- Your usage is automatically tracked, so you can see Weave's tracing features in action
+- You work with the same models and APIs you might use in production
+
+## What you'll learn
+
+In this guide, you'll:
+- Set up Weave and the W&B Inference service
+- Build a simple LLM application with automatic tracing
+- Compare multiple models
+- Evaluate model performance on a dataset
+- View your results in the Weave UI
+
+## Prerequisites
+
+Before you begin, you need:
+- A W&B account ([sign up free](https://app.wandb.ai/login?signup=true))
+- Python 3.8 or later
+- Basic Python knowledge
```

Comment:
This kind of seems obvious for our target audience, right?

Reply:
Changed the focus here to be on the required API key